### ISSUE TITLE
how can i tell if developer version of courseplay is running?

### ISSUE THREAD

**Issue by chaskuchar** [Courseplay/courseplay#1230]
fs-uk has a download for the developer version but I can't figure out where to put the exe file to update the courseplay.  it says it could not find the 'mod' file.  I ran it in the farming simulator 15 folder, in the mods folder and outside of the farming simulator folder.  result was the same in all cases.  the hud still says v4.01.0000.  thanks, chas

**Comment by gui7545** [comment]
You can find the way how to use it here : http://courseplay.github.io/courseplay/en/installation/index.html
and the link is here : https://github.com/Courseplay/courseplay/archive/master.zip
there is no exe, where did you find it ?

**Comment by chaskuchar** [comment]
fs-uk has a file to download that is labeled 'courseplay updater.  the file has three elements.  course play.exe, courseplay_updater_settings and v1.0,0.9 text file.  Added Warnings as requested by CP developers. (v1.0.0.9).  now I finally notice at the end it installs v1.0.0.0.  I downloaded courseplay-master.zip and see if I can do it correctly.  I could not find the course play master after doing the download in other downloads.  thanks, chas

Changed Entry to Courseplay_Updater_Settings.ini "Version Backups= 10 / 0 & False Disables this feature". (v1.0.0.8) (v1.0.0.8)
Added Last Run: Time - Date to log file. (v1.0.0.8)
Added compatibility to run from Task Scheduler. (v1.0.0.8)
Added more debugs to log file. (v1.0.0.8)
Added Popups on Errors, Warnings and Completion. (v1.0.0.8)
Added Entry to Courseplay_Updater_Settings.ini "Popup Messages=True\False". (v1.0.0.8)
Added Entry to Courseplay_Updater_Settings.ini "Main Directory=". (v1.0.0.8)
Renamed Settings.ini to Courseplay_Updater_Settings.ini. (v1.0.0.8)

Changed Folder Layout. (v1.0.0.7)
Fixed random temp files. (v1.0.0.7)
Added Entry to Settings.ini "Version Backups= 10". (v1.0.0.7)
Removed Backup Time & Date replaced with number. (v1.0.0.7)


Added more debugs to log file. (v1.0.0.6)
Fixed Backup Time & Date Formats. (v1.0.0.6)
Fixed Farming Simulator Running bug on Windows XP. (v1.0.0.6)

Fixed a few more Bugs. (v1.0.0.5)
7zip is no longer required to be installed. (v1.0.0.5)


Added Settings.ini to change Log file and Mod Directories. (v1.0.0.4)
Added Support for Windows Xp/Vista/7/8. (v1.0.0.4)


Backup Previous Versions of Courseplay. (v1.0.0.3)


Fixed more Bugs. (v1.0.0.2)


Fixed a few Bugs. (v1.0.0.1)


Downloads and Installs Courseplay Developer Version. (v1.0.0.0)
